src/main/resources/application.properties

```diff
@@ -2,4 +2,8 @@ logging.level.org.springframework.web=TRACE
 
 spring.data.rest.basePath=/api
 
-management.endpoints.web.exposure.include=*
\ No newline at end of file
+management.endpoints.web.exposure.include=*
+
+gaia.ldap.userDnPatterns=
```

Comment:
I would rather set this properties in the @Value annotation in this form :

```java
@Value("${gaia.ldap.userDnPatterns:}")
String userDnPatterns
```

What do you think?

Reply:
I think it's better !

Reply:
changes done :smiley: